Dockerfile

```diff
@@ -61,10 +61,11 @@ RUN --mount=type=cache,target=/go/pkg go list -mod=readonly all >/dev/null
 FROM base AS kres-linux-amd64-build
 COPY --from=generate / /
 WORKDIR /src/cmd/kres
+ARG WITH_DEBUG
 ARG VERSION_PKG="github.com/siderolabs/kres/internal/version"
 ARG SHA
 ARG TAG
-RUN --mount=type=cache,target=/root/.cache/go-build --mount=type=cache,target=/go/pkg go build -ldflags "-s -w -X ${VERSION_PKG}.Name=kres -X ${VERSION_PKG}.SHA=${SHA} -X ${VERSION_PKG}.Tag=${TAG}" -o /kres-linux-amd64
+RUN --mount=type=cache,target=/root/.cache/go-build --mount=type=cache,target=/go/pkg go build $([ "$WITH_DEBUG" == true ] && echo "-tags sidero.debug" || echo "") -ldflags "-s -w -X ${VERSION_PKG}.Name=kres -X ${VERSION_PKG}.SHA=${SHA} -X ${VERSION_PKG}.Tag=${TAG}" -o /kres-linux-amd64
```

Comment:
my eyes are bleeding... can't we do it via `Makefile` `if` ?

Reply:
Yes I don't like it either. I don't think Makefile ifs are necessarily better, but sure we can :)